python/lsst/obs/base/camera_tests.py

```diff
@@ -77,3 +84,30 @@ def test_camera_butler(self):
         self.assertEqual(camera.getName(), self.camera_data.camera_name)
         self.assertEqual(len(camera), self.camera_data.n_detectors)
         self.assertEqual(next(iter(camera)).getName(), self.camera_data.first_detector_name)
+
+    def test_plate_scale(self):
+        """Check the plate scale at center of focal plane
+
+        Check plate_scale using the FOCAL_PLANE to FIELD_ANGLE transform
+        from the camera
+        """
+        plate_scale = self.camera_data.plate_scale
+        self.assertIsNotNone(plate_scale)
+        camera = self.butler.get('camera', immediate=True)
+        focalPlaneToFieldAngle = camera.getTransformMap().getTransform(FOCAL_PLANE, FIELD_ANGLE)
+        focalPlaneRadiusMm = 0.001
```

Comment:
Here's a definite case where I wish we had astropy.quantity compatibility. I'm assuming this is to convert mm to meters?

Reply:
There is no conversion being done; focal plane coordinates are in mm (and field angle coordinates are in radians). I picked a radius small enough so as to be in the linear regime, even if the camera has significant optical distortion.

Reply:
Ah. Please make a comment to that effect. This looked like a conversion factor to me.